src/include/sof/audio/component.h

```diff
@@ -211,6 +211,7 @@ struct comp_dev {
 	uint64_t position;		/* component rendering position */
 	uint32_t frames;		/* number of frames we copy to sink */
 	uint32_t frame_bytes;		/* frames size copied to sink in bytes */
+	uint32_t output_rate;		/* 0 means all output rates are fine */
```

Comment:
I see some set to 0 in `buffer_new`, but no for this.
```c
        buffer->avail = 0;
	buffer->connected = 0;
```
But it seems we used rzalloc for all comp.

Reply:
yes, see previous comment.